Makefile

```diff
@@ -256,6 +256,10 @@ embedding: examples/embedding/embedding.cpp ggml.o llama.o common.o
 # Tests
 #
 
+benchmark: ggml.o
+	$(CXX) $(CXXFLAGS) tests/test-benchmark-q4_0-matmult.c ggml.o -o tests/test-benchmark-q4_0-matmult $(LDFLAGS)	
+	tests/test-benchmark-q4_0-matmult
+	
```

Comment:
After moving to "examples" these seem to not have been updated.
Also update the `clean:` section

Reply:
Done